_includes/nav.html

```diff
@@ -16,21 +16,21 @@
         <div class="collapse navbar-collapse" id="bs-example-navbar-collapse-1">
             <ul class="nav navbar-nav navbar-right">
                 <li>
-                    <a href="{{ site.baseurl }}/">Home</a>
-                </li>
-                <li>
                     <a href="{{ site.baseurl }}/news/">News</a>
                 </li>
                 <li>
                     <a href="{{ site.baseurl }}/publications/">Publications</a>
                 </li>
                 <li>
-                    <a href="http://oggm.readthedocs.io" target="_blank">Documentation</a>
+                    <a href="{{ site.baseurl }}/oggmev/">OGGM e.V.</a>
```

Comment:
perhaps rather "Organization" as the menu item?

Reply:
Yes, agreed